openapi-common/src/main/java/io/micronaut/openapi/SimpleSchema.java

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2017-2023 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.openapi;
-
-import java.util.Objects;
-
-import io.swagger.v3.oas.models.media.Schema;
-
-/**
- * Copy of MapSchema but without type 'object'. Need this class to correct deserializing schema without type.
- *
- * @since 4.8.7
- */
-public class SimpleSchema extends Schema<Object> {
```

Comment:
deleting a public class is a breaking change.

Reply:
Hm, This class I created, because early swagget haven't it, bu now the added `AbribtratrySchema` which the same my old `SimpleSchema` - object to serialization eqauls with the `Schema`, but, without `type` field. What do you suggest? We don't need this class anymore

Reply:
annotate it with `@Deprecated(forRemoval=true`

Reply:
done

Reply:
most of this logic was not on the public API since the openapi module is an annotation processor so not sure if this was necessary